apps/omg_watcher/lib/application.ex

```diff
@@ -19,6 +19,7 @@ defmodule OMG.Watcher.Application do
   use OMG.API.LoggerExt
 
   def start(_type, _args) do
+    DeferredConfig.populate(:omg_eth)
```

Comment:
is the ubiquitous `DeferredConfig.populate(:omg_eth)` really necessary in all cases?